plate_mapper/manual.html

```diff
@@ -236,6 +236,46 @@
+    <li>The sample name list file should contain one sample name per line. Additional information after the first tab will be trimmed off (therefore the user may use a table instead of a list).</li>
+  </ul>
+
+  <h3>(optional) <b><u>special</u></b>: special sample definition</h3>
+  <div class="table" style="height:120px">
+    <table style="width:1200px;">
+      <tr><td class="row-head"></td><td class="col-head">A</td><td class="col-head">B</td><td class="col-head" style="width:120px">C</td><td class="col-head">D</td><td class="col-head">E</td><td class="col-head">F</td><td class="col-head">G</td><td class="col-head">H</td><td class="col-head">I</td><td class="col-head">J</td><td class="col-head">K</td><td class="col-head">L</td><td class="col-head">M</td><td class="col-head">N</td><td class="col-head">O</td></tr>
+      <tr><td class="row-head">1</td><td>Code</td><td>Name</td><td>Note</td><td>ExtractionKit_lot</td><td>Extraction_robot</td><td>TM1000_8_tool</td><td>Primer_date</td><td>MasterMix_lot</td><td>Water_Lot</td><td>Processing_robot</td><td>TM300_8_tool</td><td>TM50_8_tool</td><td>Sample_Plate</td><td>Project_name</td><td>Orig_name</td></tr>
+      <tr><td class="row-head">2</td><td style="text-align:center">_</td><td>BLANK</td><td>Blank sample</td><td>PM16F14</td><td>HOWE_KF1</td><td>108379Z</td><td>1/5/2017</td><td>373139</td><td>RNBF2734</td><td>RIKE</td><td>311318B</td><td>311426B</td><td>Test_Plate_220</td><td>Test_Study</td><td>33657</td></tr>
+      <tr><td class="row-head">3</td><td style="text-align:center">+</td><td>POSCTL</td><td>Positive control</td><td>PM16F14</td><td>HOWE_KF1</td><td>108379Z</td><td>1/5/2017</td><td>373139</td><td>RNBF2734</td><td>RIKE</td><td>311318B</td><td>311426B</td><td>Test_Plate_220</td><td>Test_Study</td><td>33657</td></tr>
+      <tr><td class="row-head">4</td><td style="text-align:center">-</td><td>NEGCTL</td><td>Negative control</td><td>PM16F14</td><td>HOWE_KF1</td><td>108379Z</td><td>1/5/2017</td><td>373139</td><td>RNBF2734</td><td>RIKE</td><td>311318B</td><td>311426B</td><td>Test_Plate_220</td><td>Test_Study</td><td>33657</td></tr>
+      <tr><td class="row-head">5</td><td style="text-align:center"></td><td>EMPTY</td><td>Empty well</td><td>PM16F14</td><td>HOWE_KF1</td><td>108379Z</td><td>1/5/2017</td><td>373139</td><td>RNBF2734</td><td>RIKE</td><td>311318B</td><td>311426B</td><td>Test_Plate_220</td><td>Test_Study</td><td>33657</td></tr>
+    </table>
+  </div>
+  <ul>
+    <li>This function is to facilitate entering special types of samples. For example, when the user wants to enter a blank sample in the plate map, they simply need to type an underline (<b>_</b>). This character will be converted into "<b>BLANKx.yz</b>" in the final mapping file, in which <b>x</b>, <b>y</b> and <b>z</b> are the primer plate number, row number and column number. The metadata following the sample will be replaced with what are defined here.</li>
```

Comment:
Is "_" considered a reserved character? Is this replacement only when it is "_" in isolation?